app/src/main/java/com/kickstarter/viewmodels/EnvironmentalCommitmentsViewHolderViewModel.kt

```diff
@@ -24,21 +25,24 @@ class EnvironmentalCommitmentsViewHolderViewModel {
         fun category(): Observable<Int>
     }
 
-    class ViewModel(@NonNull val environment: Environment) :
-        ActivityViewModel<EnvironmentalCommitmentsViewHolder>(environment), Inputs, Outputs {
+    class EnvironmentalCommitmentsViewHolderViewModel() :
```

Comment:
I removed the environment dependency because we currently do not need it in this view model. If that changes in the future, we can add it back